backend/main.py

```diff
@@ -111,64 +112,10 @@ class DummyMessage(pydantic.BaseModel):
 
-                dummy_messages = [
-                    DummyMessage(
-                        task_message_id="de111fa8",
-                        user_message_id="6f1d0711",
-                        parent_message_id=None,
-                        text="Hi!",
-                        role="prompter",
-                    ),
-                    DummyMessage(
-                        task_message_id="74c381d4",
-                        user_message_id="4a24530b",
-                        parent_message_id="6f1d0711",
-                        text="Hello! How can I help you?",
-                        role="assistant",
-                    ),
-                    DummyMessage(
-                        task_message_id="3d5dc440",
-                        user_message_id="a8c01c04",
-                        parent_message_id="4a24530b",
-                        text="Do you have a recipe for potato soup?",
-                        role="prompter",
-                    ),
-                    DummyMessage(
-                        task_message_id="643716c1",
-                        user_message_id="f43a93b7",
-                        parent_message_id="4a24530b",
-                        text="Who were the 8 presidents before George Washington?",
-                        role="prompter",
-                    ),
-                    DummyMessage(
-                        task_message_id="2e4e1e6",
-                        user_message_id="c886920",
-                        parent_message_id="6f1d0711",
-                        text="Hey buddy! How can I serve you?",
-                        role="assistant",
-                    ),
-                    DummyMessage(
-                        task_message_id="970c437d",
-                        user_message_id="cec432cf",
-                        parent_message_id=None,
-                        text="euirdteunvglfe23908230892309832098 AAAAAAAA",
-                        role="prompter",
-                    ),
-                    DummyMessage(
-                        task_message_id="6066118e",
-                        user_message_id="4f85f637",
-                        parent_message_id="cec432cf",
-                        text="Sorry, I did not understand your request and it is unclear to me what you want me to do. Could you describe it in a different way?",
-                        role="assistant",
-                    ),
-                    DummyMessage(
-                        task_message_id="ba87780d",
-                        user_message_id="0e276b98",
-                        parent_message_id="cec432cf",
-                        text="I'm unsure how to interpret this. Is it a riddle?",
-                        role="assistant",
-                    ),
-                ]
+                with open(settings.DEBUG_USE_SEED_DATA_PATH) as f:
+                    dummy_messages_raw = json.load(f)
+
+                dummy_messages = [DummyMessage(**dm) for dm in dummy_messages_raw]
```

Comment:
maybe wrap this in a try block and do `logger.error` or `logger.exception` if it fails

Reply:
@yk I also thought so, but there is a broader try catch for seed data loading, so the program will still run without seed data with logging of exception and traceback if these lines got error.